docker-compose.yml

```diff
@@ -39,15 +29,13 @@ services:
       - '35729:35729'
     environment:
       - DB_1_PORT_27017_TCP_ADDR=mongodb
+      - NODE_ENV=development
 
   ## Database container
   mongodb:
     image: mongo:4.4
-    container_name: db_1
     volumes:
       - dbdata:/data/db
     restart: unless-stopped
-    ports:
```

Comment:
This isn't necessary for the trustroots service. It would be necessary for the *host* to access the db.